data/mods/Magiclysm/recipes/recipe_potions.json

```diff
@@ -0,0 +1,146 @@
+[
+  {
+    "type": "recipe",
+    "result": "potion_starter",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_POTIONS",
+    "skill_used": "cooking",
+    "skills_required": [ "spellcraft", 2 ],
+    "difficulty": 3,
+    "charges": 7,
+    "time": "120 m",
+    "batch_time_factors": [ 75, 4 ],
+    "book_learn": [ [ "alchemy_basic", 3 ] ],
+    "qualities": [ { "id": "CHEM", "level": 1 }, { "id": "CONCENTRATE", "level": 1 }, { "id": "MANA_INFUSE", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
+    "components": [
+      [
+        [ "gin", 7 ],
```

Comment:
there should be a requirement for this, i think you should use that instead.

Reply:
Eggs_bird doesn't respect mass of each egg being differrent, it counts each egg as 1 regardless of size.

Reply:
Changed to hard_liquor.